03-feedback.md

```diff
@@ -0,0 +1,73 @@
+
+##Anatomy of Feedback
+
+The practice of giving feedback can be incredibly powerful in its ability to expose things we couldn't perceive in ourselves in the intensity of the stage, but it can also be a very fraught process. Criticism can challenge our self image, and chafe on our insecurities and doubts; receiving criticism can feel like a place of powerlessness. If we are to successfully critique each other, we must find a way to speak our minds in a way each other can hear - how do we criticize each other in a way designed to empower and not to tear down? Giving good criticism consists of three cardinal rules:
+
+ - **Be specific.** By keeping our feedback specific, we circumscribe its scope, which accomplishes two things. First, it gives the subject a clear idea of exactly where problems lie; if feedback is too nebulous, it becomes impossible to address. Second, articulating exactly what needs attention helps the subject to separate the thing being critiqued from their practice as a whole, and adds distance between the criticism and the individual's identity. Broad, vague criticisms feel like they are about us; specific criticisms feel like they are about our practice.
+
+ In addition to keeping criticism specific, keep the list short, too - pick the biggest problem to call out, and leave the rest for next time. If your subject can fix one thing every time they teach, they will reach mastery plenty fast.
+
+ - **Be actionable.** Actionable feedback is feedback that comes with some clear ideas on exactly what to do in order to improve. By giving a clear roadmap for addressing the problem, we help put the power back in our subject's hands, and build up their sense of agency; now instead of a giving a criticism that feels like an inescapable condemnation, we've given a few short and manageable steps to overcoming the issue we called out, and becoming an even stronger instructor.
+
+ - **Be polite.** While it seems trite in writing, social gaffes are never far away when giving criticism. A critical conversation often can't help but exceed the usual boundaries of frankness and honesty; best then to be especially mindful of framing comments in a positive light, describing opportunities to improve, and not going further than we must.
+
+[Exercise 1: Basic Feedback](http://mozillascience.github.io/instructorTraining/feedback/feedback.html)
+
+##Structured Feedback
+
+Trying to articulate exactly where your fellow instructor should turn her attention to in order to improve can be a nebulous and difficult task in practice. Things can go much more smoothly with a bit of structure to the process.
```

Comment:
“turn her attention to in order” → “turn her attention in order”.  No big deal, but I think it reads more clearly that way.